spec/indico_batch_spec.rb

```diff
@@ -31,8 +31,9 @@
     expect(Set.new(response[1].keys)).to eql(expected_keys)
   end
 
-  it 'should tag text with correct sentiment tags' do
-    response = Indico.batch_sentiment(['Worst movie ever.'], @config)
+  it 'should tag text with correct posneg tags' do
+    data = ['Worst movie ever.', 'Best movie ever']
+    response = Indico.batch_posneg(data)
```

Comment:
Ah, I had it backwards. It's sentiment now, not 'posneg'. I was thinking we don't need to test for 'batch_sentiment' since we made the switch from sentiment to posneg before we offered batch, but thats backwards. I'll fix it.